### ISSUE TITLE
Improve readability of "green" font on the bank scene by using darker shade in piggy bank

### ISSUE THREAD

**Issue by kathy-phet** [phetsims/number-line-integers#44]
On the phone, the contrast of the green font on white is hard to read. Can we use the darker shade of green in the piggy bank? (the upper text box, versus the current shade which is in the lower text box). The red font has really good contrast and is easy to read.
![image](https://user-images.githubusercontent.com/5096804/67601848-957e0400-f732-11e9-807f-b42cbf433b70.png)

**Comment by jbphet** [comment]
This doesn't seem too contraversial, so I'm going to implement it on my next NLI development day (though @amanda-phet is of course free to chime in if there are objections).

**Comment by amanda-phet** [comment]
That works for me. 

@jbphet you have the hex code, right? I think I gave it to you as part of the color scaling from lightest to darkest. But let me know if you need it.

**Comment by jbphet** [comment]
> @jbphet you have the hex code, right?

Not that I know of.  The darker color is actually generated as a combination of the nominal fill and the overlayed image that contains the flowers or lightning bolt, so it's not actually used in the code anywhere.  To get the value, I just extracted it from a screenshot of the piggy bank and used the coler from the dark patch just below the eye.  The value is `#0e977b`, and that matches the text in @kathy-phet's example, so I went with that number.  It looks good to me, and I don't think anyone is likely to object, so I'm going to close.
